ingestion/setup.py

```diff
@@ -233,6 +233,7 @@
     "mysql": {VERSIONS["pymysql"]},
     "nifi": {},  # uses requests
     "okta": {"okta~=2.3"},
+    "openlineage": {"confluent-kafka"},
```

Comment:
```suggestion
    "openlineage": {*COMMONS["kafka"]},
```

defining just `confluent-kafka` here would conflict with the version of `confluent-kafka` required for kafka connector itself, hence it would be better if we just use the common kafka dependencies which are already available!